packages/api-core/src/domain-suggestions/types.ts

```diff
@@ -181,6 +183,17 @@ export interface DomainSuggestion {
 	 * Whether the client should show the dot gay notice
 	 */
 	dot_gay_notice_required?: true;
+
+	/**
+	 * Policy notices associated with the domain
+	 */
+	policy_notices?: PolicyNotice[];
```

Comment:
Now that we have the policy notices as an array, do we want to remove `hsts_required` and `dot_gay_notice_required`?